manim/_config/main_utils.py

```diff
@@ -111,7 +111,6 @@ def parse_args(args: list) -> argparse.Namespace:
     elif subcmd == "plugins":
         return _parse_args_plugins(args)
     elif args[1] == "--version":
-        print(f"Manim Community v{ __version__ }")
```

Comment:
I think we should either not print out the version in __main__.py, or we should do a sys.exit()